<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPacket</class>
 <widget class="QDialog" name="ControlPacket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>除尘:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>开</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>关</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>开</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>关</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>运行/维护：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_4">
       <item>
        <property name="text">
         <string>运行</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>维护</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>本地</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>远程</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>开/关门：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>本地/远程：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>面阵开/关:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_5">
       <item>
        <property name="text">
         <string>开</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>关</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>散热开/关：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_7">
       <item>
        <property name="text">
         <string>开</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>关</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>线阵开/关：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_8">
       <item>
        <property name="text">
         <string>开</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>关</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_6">
       <item>
        <property name="text">
         <string>开</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>关</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>加热开/关：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>系统工作状态：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>模拟测试触发指令：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_9">
       <item>
        <property name="text">
         <string>测试开</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>测试关</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_10">
       <item>
        <property name="text">
         <string>0-就绪</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1-车辆进入状态</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2-维护状态</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3-模拟运行状态</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4-自检状态</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
